### PR TITLE
Aw 2 refactoring

### PR DESCRIPTION
**Pull request by asilmb**

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -7,8 +7,9 @@ RewriteRule ^assets/(.*)$ /web/assets/$1 [L]
...
 RewriteRule ^js/(.*)$ web/js/$1 [L]
 RewriteRule ^images/(.*)$ web/images/$1 [L]
-RewriteRule (.*) /web/$1
+RewriteRule ^(.+)?$ /web/$1
 
 RewriteCond %{REQUEST_FILENAME} !-f
 RewriteCond %{REQUEST_FILENAME} !-d
-RewriteRule . /web/index.php
\ No newline at end of file
+RewriteRule . /web/index.php
```
Comment (asilmb) [inline .htaccess:14]: 345

Comment (Madison2201) [inline .htaccess:14]: 567

**Review by asilmb — COMMENTED** [review]
456

File: controllers/AnkiController.php [hunk 2]
```diff
@@ -2,50 +2,82 @@
...
-            $mail->mail = $model->email;
-            $mail->hash = hash("md5",$model->email);
-            if($mail->save()){
-                $this->sentEmail($mail);
-                return $this->goHome();
-            }
-
+            $mail->mail = $model->mail;
+            $mail->hash = hash("md5", $model->mail);
+            $mail->save();
```
Comment (asilmb) [inline controllers/AnkiController.php:33]: proverka chto true

**Pull request closed without merging**